formats/syslog/syslog.go

```diff
@@ -73,14 +73,30 @@ func SampleData() TemplateData {
 }
 
 func RandomData() TemplateData {
+	limit := viper.GetInt("message.max-random-cap")
 	return TemplateData{
-		Facility: randomdata.Number(0, 24),
-		severity: randomdata.Number(0, 8),
+		Facility: randomdata.Number(0, cap(24, limit)),
+		severity: randomdata.Number(0, cap(8, limit)),
```

Comment:
Note: Facility and severity have semantics, I don't think they should be controlled by a global random cap.
(They are also low-cardinality).